student/src/Form/StudentSignUpForm.php

```diff
@@ -0,0 +1,177 @@
+  * php version 7.2.24
+  * 
+  * @category Custom_Form
+  * @package  Student
+  * @author   Neil Bridges <neil@discoverygarden.com>
+  * @license  test.com  test
+  * @link     test
+  * @return   /custom/student/src/Form/StudentSignUpForm.php
+  */  
+ 
+namespace Drupal\student\Form;
+
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Session\AccountInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Implements an example form.
+ *    
+ * Build Form Description
+ *
+ * @category Student
+ * @package  Student
+ * @author   Neil Bridges <neil@discoverygarden.com>
+ * @license  student
+ * @link     Student 
+ * @return   /custom/student/src/Form/StudentSignUpForm.php      
+ */ 
+class StudentSignUpForm extends FormBase
+{
+        
+    protected $account;
+
+    /**
+     * Class constructor.
+     * Build Form Description
+     *
+     * @category Student
+     * @package  Student
+     * @author   Neil Bridges <neil@discoverygarden.com>
+     * @link     Student 
+     * @license  student  
+     * @return   /custom/student/src/Form/StudentSignUpForm.php      
+     */    
+ 
+    public function __construct(AccountInterface $account) 
+    {
+        $this->account = $account;
+    }
+
+    /** 
+     * Build Form Description
+     *
+     * @category Student
+     * @package  Student
+     * @author   Neil Bridges <neil@discoverygarden.com>
+     * @link     Student 
+     * @license  student  
+     * @return   /custom/student/src/Form/StudentSignUpForm.php      
+     */
+    public static function create(ContainerInterface $container) 
+    {
+        // Instantiates this form class.
+        return new static(
+            // Load the service required to construct this class.
+            $container->get('current_user')
+        );
+    }
+
+    /** 
+     * Build Form Description
+     *
+     * @category Student
+     * @package  Student
+     * @author   Neil Bridges <neil@discoverygarden.com>
+     * @link     Student 
+     * @license  student  
+     * @return   /custom/student/src/Form/StudentSignUpForm.php      
+     */
+    public function getFormId() 
+    {
+        return 'student_sign_up_form';
+    }
+
+   
+    /** 
+     * Build Form Description
+     *
+     * @category Student
+     * @package  Student
+     * @author   Neil Bridges <neil@discoverygarden.com>
+     * @link     Student 
+     * @license  student 
+     * $container student
+     * $form student
+     * $form_state student           
+     * @return   /custom/student/src/Form/StudentSignUpForm.php      
+     */
+    public function buildForm(array $form, FormStateInterface $form_state) 
+    {
+
+        // Get current user data.
+        $uid = $this->account->id();
+        $current_user = $this->account->getAccountName();
+                
+  
+          $form['uid'] = array(
+          '#type' => 'hidden',
```

Comment:
Things inside of an array defined like this should be tabbed further in as well as using the array short-form instead of calling `array()`. Also, boolean values should be capitalized. So for example:

```suggestion
          $form['uid'] = [
              '#type' => 'hidden',
              '#title' => t('User ID:'),
              '#default_value' => "$uid",
              '#required' => TRUE,
          ];
```